docs/concepts/demisto-sdk.md

```diff
@@ -271,10 +271,17 @@ and the `DEMISTO_API_KEY` prior to running this command in order to establish a
  
  - Upload a pack to the server without certificate validation:
     ```buildoutcfg
-    demisto-sdk -i Pack/myPack --insecure
+    demisto-sdk upload -i Pack/myPack --insecure
     ```
    This iterates over all the content entities in the pack `myPack` and uploads them to the preset XSOAR server 
    without checking the certification. Note that this command does not upload the entities as a whole pack but instead uploads them individually.
+   
+   
+   - Zip and upload a whole pack to the marketplace:
+    ```buildoutcfg
+    demisto-sdk upload -i Pack/myPack -z
+    ```
+   This will zip the pack `myPack` and uploads it to the marketplace in the preset XSOAR server.
```

Comment:
```suggestion
   This will zip the pack `myPack` and upload it to the marketplace in the preset XSOAR server.
```

Reply:
let's add an explanation about the fact that if you will try and upload preexisting content it will ask you if you want to overwrite it.